qtaskbarcontrol.h

```diff
@@ -53,6 +54,7 @@ public slots:
 
 private:
 	QScopedPointer<QTaskbarControlPrivate> d;
+	QPointer<QWidget> _watchedWidget;
```

Comment:
Please move this to QTaskbarControlPrivate.

Reply:
Oh yes my way breaks pimpl